assignment/src/com/assignmentweek4/springwithxml/MyApp.java

```diff
@@ -0,0 +1,11 @@
+package com.assignmentweek4.springwithxml;
+
+public class MyApp {
+
+	public static void main(String[] args) {
+			//Coach theCoach = new TrackCoach();
+		//	System.out.println(theCoach.getDailyWorkout());
+			
+	}
+
+}
```

Comment:
Delete if this class is not required as all the code inside main method is commented.